workbench_access.module

```diff
@@ -168,6 +168,35 @@ function workbench_access_node_access(NodeInterface $node, $op, AccountInterface
 }
 
 /**
+ * Implements hook_node_create_access().
+ *
+ * @link https://www.drupal.org/node/2348203
+ */
+function workbench_access_node_create_access(AccountInterface $account, $context, $entity_bundle) {
+  $return = AccessResult::neutral();
+  // Check that our access control is configured.
+  $manager = \Drupal::service('plugin.manager.workbench_access.scheme');
+  $scheme = $manager->getActiveScheme();
+  if (!$scheme) {
```

Comment:
Actually, I'm always uncertain on the negation syntax in that IF case.

It's not `if ($scheme 1= $manager->getActiveScheme())` and I don't trust `if (!$scheme = $manager->getActiveScheme())` to read clearly.